tests/foreman/ui/test_settings.py

```diff
@@ -180,12 +179,12 @@ def test_positive_host_dmi_uuid_duplicates(session, setting_update):
 
     :CaseImportance: High
     """
-    property_value = f'[ {gen_string("alpha")} ]'
+    property_value = gen_string("alpha")
     property_name = setting_update.name
     with session:
         session.settings.update(f'name = {setting_update.name}', property_value)
         result = session.settings.read(f'name = {property_name}')
-        assert result['table'][0]['Value'] == property_value
+        assert result['table'][0]['Value'][1:-1] == property_value
```

Comment:
@jyejare There is a square bracket which needs to be removed.

Reply:
@shweta83 you should use `rstrip(']')` to transform the string for something like this.

If the appropriate syntax was list slicing, in-line comments are always a good idea for things like this, because there's no telling what was meant to be stripped, if at some point this assertion fails.

Is that square bracket in the string in the UI, or is it coming from the table read?

Reply:
@mshriver UI returns list but table list changes it to sting. strip() is added to remove the brackets here.

Reply:
trigger: test-robottelo
pytest: tests/foreman/ui/test_settings.py -k test_positive_host_dmi_uuid_duplicates